recipes/django-parler/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "django-parler" %}
+{% set version = "2.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f633522c06727852978cf2f92c94a98d900dff47235412e64ed46d63e68b1f09
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  requires:
+    - six
+    - django
+  commands:
+    - python -c "from django.conf import settings; settings.configure(); import parler"
+
+about:
+  home: https://pypi.org/project/django-parler
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: 'Simple Django model translations without nasty hacks.'
+
+  description: |
+    Features:
+      - Nice admin integration
+      - Access translated attributes like regular attributes.
+      - Automatic fallback to the default language.
+      - Separate table for translated fields, compatible with django-hvad.
+      - Plays nice with others, compatible with django-polymorphic, django-mptt and such:
+        - No ORM query hacks.
+        - Easy to combine with custom Manager or QuerySet classes.
+        - Easy to construct the translations model manually when needed.
+  
+  doc_url: https://django-parler.readthedocs.io/
+  dev_url: https://github.com/mbi/django-parler/
+
+extra:
+  recipe-maintainers:
+    - django-parler
```

Comment:
Is this a real user or team?

Reply:
It's a team. You may remove it if teams aren't allowed to be maintainers. Or should I go ahead and make the modification myself?

Thanks @scopatz

Reply:
Teams can be maintainers, but I think this will resolve to the conda-forge team in its current form

Reply:
alright, I'll be sole maintainer